examples/contour/01-crds.yaml

```diff
@@ -6975,7 +6975,7 @@ spec:
                       type: string
                     targetNamespaces:
                       description: required, the namespaces the authority to reference
-                        the the secret will be delegated to. If TargetNamespaces is
+                        the secret will be delegated to. If TargetNamespaces is
```

Comment:
You will need to make this change in the `tlscertificatedelegation.go` file in `apis/projectcontour/v1`, and then run `make generate to ensure all of the generated YAML is in sync with it.

Reply:
Still need to run `make generate` and commit the results..

Reply:
Hmm, not sure what error you're getting. Is it OK if I push the update to your branch?

Reply:
Done